GameFiles/GameInterface/_integration_iw4x.gsc

```diff
@@ -330,71 +333,57 @@ NoClipImpl()
     else
     {
         self SetClientDvar( "sv_cheats", 1 );
-        self SetClientDvar( "cg_thirdperson", 1 );
+        self SetClientDvar( "cg_thirdperson", 0 );
         self SetClientDvar( "sv_cheats", 0 );
         
         self God();
         self Noclip();
-        self Hide();
+        self Show();
         
         self.isNoClipped = false;
         
         self IPrintLnBold( "NoClip disabled" );
     }
-
-    self IPrintLnBold( "NoClip enabled" );
 }
 
 HideImpl()
 {
     if ( !IsAlive( self ) )
     {
         self IPrintLnBold( "You are not alive" );
-        return;
```

Comment:
why did we remove this return? Do we actually want to run the logic if the player is not alive?

Reply:
whoops